pkg/resources/sop_url.go

```diff
@@ -35,4 +35,6 @@ const (
 	SopUrlThreeScaleDeveloperUIBBT                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleDeveloperUIBBT.asciidoc"
 	SopUrlThreeScaleSystemAdminUIBBT                          = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleSystemAdminUIBBT.asciidoc"
 	SopUrlPodDistributionIncorrect                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/multi-az/pod_distribution.md"
+	SopUrlSloRhssoAvailabilityAlert                           = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/SloRhssoAvailabilityAlert.md"
```

Comment:
I think it would be worth looking at the MAS SSO SOP also and seeing do we want to take any of that and apply it here